### ISSUE TITLE
WebSockets crashes server in some instances

### ISSUE THREAD

**Issue by marclundgren** [blenderskool/blaze#48]
```
curl 'https://blaze.unubo.app/' -H 'authority: blaze.unubo.app' -H 'pragma: no-cache' -H 'cache-control: no-cache' -H 'upgrade-insecure-requests: 1' -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36' -H 'sec-fetch-user: ?1' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3' -H 'sec-fetch-site: cross-site' -H 'sec-fetch-mode: navigate' -H 'accept-encoding: gzip, deflate, br' -H 'accept-language: en-US,en;q=0.9' -H 'cookie: __cfduid=dd4671b66d49ae62acba3c9c6bd5a637b1574100693' --compressed
```

**Comment by marclundgren** [comment]
If it helps, I'm currently on a starbucks (public) network.

**Comment by blenderskool** [comment]
Thanks, I have identified this bug recently. It occurs due to a bug in WebSockets implementation causing the server to crash in some instances.

**Comment by blenderskool** [comment]
This was fixed in the v2.0.0 release and does not happen anymore 🙂
